modules/sdk-coin-atom/test/unit/transactionBuilder/StakingDeactivateBuilder.ts

```diff
@@ -0,0 +1,27 @@
+import { getBuilderFactory } from '../getBuilderFactory';
+import * as testData from '../../resources/atom';
+import should from 'should';
+import { toHex, TransactionType } from '@bitgo/sdk-core';
+import { Transaction as AtomTransaction } from '../../../src/lib/transaction';
+import { fromBase64 } from '@cosmjs/encoding';
+
+describe('Atom Undelegate txn Builder', () => {
```

Comment:
Same for undelegate, if you can make an integration test, that would be great

Reply:
done